logstash-core/src/test/java/org/logstash/ext/JrubyEventExtLibraryTest.java

```diff
@@ -43,6 +43,18 @@ public void shouldSetJavaProxy() throws IOException {
         }
     }
 
+    @Test
+    public void correctlyHandlesNonUnicodeKeys() {
```

Comment:
Could you rename this test case to indicate that it's testing for characters with more than two-byte representations?

Reply:
@danhermann done, renamed to `*nonAscii*` since it also failed with `ü` and such (sorry for the confusion here earlier ... same as with the PQ serialisation effectively all non-ascii were broken here).